shuffle.js

```diff
@@ -0,0 +1,13 @@
+function shake(){
```

Comment:
`shake` is meant to take the `board` as an argument and insert random letters into the `board` at the appropriate positions.